gpumon/gpumon_server.ml

```diff
@@ -51,22 +51,22 @@ module Make(I: Interface) = struct
           compat
         else
           let msg = Printf.sprintf
-            "%s: pGPU host driver version %d < %d does not support migration"
-            this major host_driver_supporting_migration in
-          raise (Gpumon_interface.NvmlFailure msg)
+              "%s: pGPU host driver version %d < %d does not support migration"
+              this major host_driver_supporting_migration in
+          raise Gpumon_interface.(Gpumon_error (NvmlFailure msg))
       with
-        | Gpumon_interface.NvmlFailure(_) as err -> raise err
-        | err -> raise (Gpumon_interface.NvmlFailure (Printexc.to_string err))
+      | Gpumon_interface.(Gpumon_error (NvmlFailure _)) as err -> raise err
+      | err -> raise Gpumon_interface.(Gpumon_error (NvmlFailure (Printexc.to_string err)))
 
     let get_vgpu_metadata _ dbg domid pgpu_address =
       let interface = get_interface_exn () in
       let domid'    = string_of_int domid  in
       try
         Nvml.device_get_handle_by_pci_bus_id interface pgpu_address
         |> fun device -> Nvml.get_vgpus_for_vm interface device domid'
-        |> List.map (Nvml.get_vgpu_metadata interface)
+                         |> List.map (Nvml.get_vgpu_metadata interface)
```

Comment:
Indentation here is weird

Reply:
For some reason ocp-indent does that unless I parenthesise the function above, then it indents correctly.